src/components/checkboxFilter/index.tsx

```diff
@@ -18,34 +18,55 @@ function CheckboxFilter<ItemKey extends string, FilterName extends string>({
   const hasGroups = items.some(
     (item) => (item.childCheckboxes ?? []).length > 0,
   );
+
+  const itemsPerColumn = Math.ceil(items.length / numberOfColumnsOnDesktop);
+
+  // Create an array to store items grouped by columns
+  const columns = Array.from(
+    { length: numberOfColumnsOnDesktop },
+    (_, columnIndex) => {
+      const columnStartIndex = columnIndex * itemsPerColumn;
+      const columnEndIndex = Math.min(
+        (columnIndex + 1) * itemsPerColumn,
+        items.length,
+      );
+      return [...items].slice(columnStartIndex, columnEndIndex);
+    },
+  );
+
   return (
     <TranslationProvider language={language} scopes={['checkboxFilter']}>
       <Box
         sx={{
-          columns: { md: numberOfColumnsOnDesktop, base: 1 },
-          columnRule: 'solid var(--chakra-colors-gray-100) 1px',
-          columnGap: 'var(--chakra-space-4xl)',
+          display: 'grid',
```

Comment:
I think you accidentally missed to add the gray separator we had before `columnRule: 'solid var(--chakra-colors-gray-100) 1px',`